tests/cypress/integration/actions_tasks_objects/case_53_object_propagate.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Object propagate.', () => {
+    const caseId = '53';
+    const createCuboidShape2Points = {
+        points: 'From rectangle',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+    let maxFrameNumber = 0;
+    const propagateOnOneFrame = 1;
+    const propagateOnTwoFrame = 2;
+
+    function startPropagation() {
+        cy.get('#cvat-objects-sidebar-state-item-1').find('[aria-label="more"]').trigger('mouseover');
+        cy.get('.cvat-object-item-menu').within(() => {
+            cy.contains('button', 'Propagate').click();
+        });
+    }
+
+    before(() => {
+        cy.openTaskJob(taskName);
+        cy.get('.cvat-player-last-button').click();
+        cy.get('.cvat-player-frame-selector')
+            .find('input')
+            .then((frameSelector) => {
+                maxFrameNumber = Number(frameSelector.val());
+            });
+        cy.get('.cvat-player-first-button').click();
+        cy.createCuboid(createCuboidShape2Points);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('On the 1st frame propagate object on 1 frame.', () => {
+            startPropagation();
+            cy.get('.cvat-propagate-confirm-object-on-frames')
+                .find('input')
+                .should('have.attr', 'value', maxFrameNumber)
```

Comment:
Let's remove this check. It is not actually true because default it set to 50.

Reply:
No problem. Done.